indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -282,6 +282,36 @@ $i-box-padding: 10px;
     }
 }
 
+.i-box {
```

Comment:
This is intended. I think this makes easier to understand that `@include transition` is there because of the `.header-only` set of rules.